aca_movie.pl

```diff
@@ -131,16 +132,41 @@
       my $dbh = sql_connect('sybase-aca-aca_read');
       my @obsinfo = sql_fetchall_array_of_hashref($dbh,
                                                   "select * from observations where obsid = $opt{obsid}");
+      my $start;
+      my $stop;
       if (not scalar(@obsinfo)){
-          croak("-obsid specified but not found in observations_all sybase table");
+          my $obs_str = sprintf("%05d", $opt{obsid});
+          my $obs_top_dir = substr($obs_str, 0, 2);
+          my @obs_dirs = sort(glob("${MICA_ASP1}/${obs_top_dir}/${obs_str}_*"));
+          if (scalar(@obs_dirs)){
+              $obs_dir = $obs_dirs[-1];
+              print "using $obs_dir of ASP1 for tstart/tstop \n" if $opt{loud}; 
+          }
+          else{
+              croak("-obsid specified but directory not found in ${MICA_ASP1}/${obs_top_dir} or in observations_all sybase table");
+          }
+          # Get a list of L1 ACA image files
+          my $asp1_dir = $obs_dirs[-1];
+          my $l1_glob = "$asp1_dir/pcad*asol1.fits*";
+          my @files = glob($l1_glob);
+          croak("no asol files found for obsid in $asp1_dir") unless @files;
+          # Set reasonable value of tstart based on first file name timestamp
```

Comment:
Oops, this comment isn't true.  I gave up on the regex timestamp thing and just used the header.